chromium/README.md

```diff
@@ -8,6 +8,8 @@ Further be aware that both cloning the source code as well as building Chromium
 
 2. Navigate to the root directory of the source code, which we will refer to hereafter as `<CHROMIMUM_ROOT>`, and run `git checkout 85.0.4161.2`, which is the latest tag for which we have verified the build instructions. Then, to ensure that all of chromium's third party dependencies are compatible with this tag, run `gclient sync`.
 
+*Note*: Depending on the OS version installed, you may have to install python2, e.g., using `sudo apt install -y python2`, and ensure it's set as the system default, e.g., via `sudo ln -s /usr/bin/python2 /usr/bin/python`.
```

Comment:
Really? Is it Chromium that uses Python2?  Because I thought all the Python we've been writing has required Python3.

Reply:
I also was surprised: Yes, it's the chromium build system (at version 85.0.4161.2) that has this dependency -- it was uncovered when switching to Ubuntu 20 -- which doesn't have python2 by default any more. The cleanest way to get rid of this note is to upgrade to a more current chromium version that shouldn't have this dependency any more (#74)